### ISSUE TITLE
"About Rstudio" misplaced in French 

### ISSUE THREAD

**Issue by romainfrancois** [rstudio/rstudio#11189]
The "Apropos de RStudio" menu appears twice in "Aide" in the French ui:

<img width="347" alt="image" src="https://user-images.githubusercontent.com/2625526/168025324-1ffdf84e-fba6-45b9-ae96-e457f63d9ab1.png">

as opposed to once in the "RStudio" and once in the "Help": 

<img width="207" alt="image" src="https://user-images.githubusercontent.com/2625526/168024884-e14f60be-9cac-4cee-8cd4-3d29cccd2429.png">

<img width="369" alt="image" src="https://user-images.githubusercontent.com/2625526/168024947-1e806b32-4a37-4b3f-a411-f64a0ef84b78.png">

**Comment by romainfrancois** [comment]
RStudio 2022.06.0-daily+391 "Spotted Wakerobin"

**Comment by mikebessuille** [comment]
@romainfrancois can you please test this in the latest SW?

**Comment by romainfrancois** [comment]
I did test with 421, which is the most recent from https://dailies.rstudio.com/rstudio/spotted-wakerobin/desktop/macos/# but it's before the pr.

**Comment by gtritchie** [comment]
Yes, there's a problem with the build pipeline, haven't had a new Mac Desktop (Qt) build for several days.

**Comment by romainfrancois** [comment]
Testing with RStudio 2022.06.0-daily+461 "Spotted Wakerobin" Daily. 

✅ Correctly seeing "Aprops de RStudio" in the "Aide" menu: 

<img width="447" alt="image" src="https://user-images.githubusercontent.com/2625526/171353417-228c84cb-6d5b-400d-8511-f7a18631c71b.png">

❌ But not in the "RStudio" menu: 

<img width="372" alt="image" src="https://user-images.githubusercontent.com/2625526/171353477-02fd76e0-8594-4b1b-8de2-a308cd63e604.png">

✅ as opposed to the English version: 

<img width="183" alt="image" src="https://user-images.githubusercontent.com/2625526/171353554-a1a465ab-a7df-4b06-afeb-451e46493e01.png">

❌ Still seeing the "Preferences ... " in "Editer": 

<img width="389" alt="image" src="https://user-images.githubusercontent.com/2625526/171353867-97c87d72-f559-4c80-93bd-6de29104e9ab.png">

❌  Seeing "Global options ... " twice in English: 

<img width="324" alt="image" src="https://user-images.githubusercontent.com/2625526/171354048-460006b3-d054-445f-b198-7958fe6ed7aa.png">

✅  "Outils" has options items. 

<img width="389" alt="image" src="https://user-images.githubusercontent.com/2625526/171354206-e96ab27c-fb02-4f4d-9712-4b15ca7a2725.png">

✅ I can see the "Window" menu, although not localized: 

<img width="401" alt="image" src="https://user-images.githubusercontent.com/2625526/171354705-55d15821-b155-4bff-bf1f-1ff75bd35af9.png">

**Comment by gtritchie** [comment]
In my PR I said:

> Not fixed (and I don't think we'll ever fix these for Qt desktop):
> 
> About RStudio not displayed on RStudio menu in French (it's available on Help/Aide menu and via command-palette so I don't think this is a major problem)
> The "Editer" menu still has a "Prefeferences..." item at the end that isn't there in English; not a huge problem

The duplicated Global Options in English is unexpected, however, so I'll check on that.
